src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -5,30 +5,38 @@
 import java.util.*;
 
 public class LottoTicket {
-    private static final int SIZE=6;
+    private static final int EMPTY_SIZE = 0;
+    private static final int SIZE = 6;
     private final List<LottoNumber> values;
 
     private LottoTicket(List<LottoNumber> values) {
         validation(values);
         this.values = values;
     }
 
-    private void validation(List<LottoNumber> values) {
-        if (values.size() != SIZE) {
-            throw new LottoTicketSizeException("Lotto ticket can have only six lotto numbers");
-        }
-    }
-
     public static LottoTicket create(List<LottoNumber> values) {
+        Collections.sort(values);
         return new LottoTicket(Collections.unmodifiableList(new ArrayList<>(values)));
     }
 
+    public boolean hasThisNumber(LottoNumber lottoNumber) {
+        return this.values.contains(lottoNumber);
+    }
+
+    public int howManyMatch(LottoTicket lottoTicket) {
+        return (int) this.values.stream()
+                .filter(lottoTicket::hasThisNumber)
+                .count();
+    }
+
     protected int size() {
         return this.values.size();
     }
 
-    public List<LottoNumber> getValues() {
-        return Collections.unmodifiableList(new ArrayList<>(this.values));
+    private void validation(List<LottoNumber> values) {
+        if (values.size() != EMPTY_SIZE && values.size() != SIZE) {
```

Comment:
null이 들어올 수도 있기 때문에 NPE 방지를 위해 null check를 우선순으로 해주는게 좋아보여요 :)
```suggestion
        if (values == null || values.isEmpty() || values.size() != SIZE) {
```